example/server/integrations.py

```diff
@@ -783,77 +783,82 @@ def toml_integration():
 
 
 def scripts_integration(page_content: Optional[Dict]):
```

Comment:
If we're going with the async/sync per-script functionality, we need to adjust the return type. Make sure you update the `scripts()` default javascript function with the new return type. The return type should be able to communicate the async/sync thing for now, but allow for future attributes to be added. (I'm thinking an object)